RequestService/RequestService.Core/Services/IJobService.cs

```diff
@@ -10,8 +10,8 @@ namespace RequestService.Core.Services
 {
     public interface IJobService
     {
-        Task<List<JobSummary>> AttachedDistanceToJobSummaries(string volunteerPostCode, List<JobSummary> jobSummaries, CancellationToken cancellationToken);
-
+        Task<List<JobSummary>> AttachedDistanceToJobSummaries(string volunteerPostCode, List<JobSummary> jobHeaders, CancellationToken cancellationToken);
```

Comment:
Rename AttachedDistanceToJobSummaries and AttachedDistanceToJobSummaries to reference "Headers"?